AncientReligions/localisation/ACR_cybrxkhan_misc.csv

```diff
@@ -53,19 +53,32 @@ ancrel.4160.a;Terrible! To think he murdered a cat, even by accident! An unforgi
 ancrel.4160.b;Come now, it's just a cat.;;;;;;;;;;;;;x
 VIET_stupid_cat_lovers_boooo;xxxxx;;;;;;;;;;;x
 ancrel.4161.desc;The ancient Egyptians put anyone who killed a cat - even accidentally - to death immediately. Their justice was swift and vengeful, and so is yours. The cat is sacred, are they not, particularly to the great Goddess Bast? May the criminal’s blood be adequate satisfaction for the great Goddess.;;;;;;;;;;;x
-ancrel.4161.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4162.desc;xxxxx;;;;;;;;;;;x
-ancrel.4162.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4163.desc;xxxxx;;;;;;;;;;;x
-ancrel.4163.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4164.desc;xxxxx;;;;;;;;;;;x
-ancrel.4164.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4165.desc;xxxxx;;;;;;;;;;;x
-ancrel.4165.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4166.desc;xxxxx;;;;;;;;;;;x
-ancrel.4166.a;Xxxxxxffff;;;;;;;;;;;;;x
-ancrel.4167.desc;xxxxx;;;;;;;;;;;x
-ancrel.4167.a;Xxxxxxffff;;;;;;;;;;;;;x
+ancrel.4161.desc;Ancient texts describe how in old Egypt, women were viewed as having greater amount of rights, privileges, and opportunities than they do today. Or so I've been told. Perhaps I should treat the women of my court with more respect if I am to properly emulate the customs of ancient Egypt.;;;;;;;;;;;x
```

Comment:
ancrel.4161.desc is in here twice, with different descriptions.
Could you figure out how this should go, @cybrxkhan ?